packages/documentation-framework/app.js

```diff
@@ -24,6 +24,12 @@ const AppRoute = ({ child, title, path }) => {
       'page_title': (title || pathname)
     });
   }
+
+  if(pathname.startsWith("/v4")){
```

Comment:
Maybe we can add a comment along the lines of "Redirect all v4 paths to the archived v4 site". And will we always need this? Wondering how long we plan on hosting that archived site. If temporary, we can add a TODO to clean this up.

Reply:
I think we'll want to host the v4 archive at least until v6, so it's a fair bit away if not longer. I'm leaving out the temporary blurb for now unless @evwilkin do you know how long we plan to keep the v4 site?

Reply:
yeah we still host a v3 site for PF, too. The archive site will be around for a while

Reply:
Agree with Nicole - no current plans for when we'd stop hosting v4 docs